PaddleAudio/paddleaudio/models/wav2vec2/activations.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
We just need to add Paddle Author outside the original license. So PaddlePaddle Author should be in the first line.